libraries/abstractions/pkcs11/test/iot_test_pkcs11.c

```diff
@@ -44,6 +44,15 @@
 #include "iot_test_pkcs11_config.h"
 #include "mbedtls/x509_crt.h"
 
+/* Model-based testing includes. */
```

Comment:
You can remove these includes

Reply:
The test cases will be picked up without them

Reply:
Removed with r7272131.